### ISSUE TITLE
Using master branch with npm in cytoscapejs

### ISSUE THREAD

**Issue by cmdcolin** [tgdwyer/WebCola#165]
I was trying to see if I could install webcola from NPM to use in cytoscape, but I get the error that cola.adaptor is not a function

```
var webcola = require('webcola');
var cy = require('cytoscape');
var cycola = require('cytoscape-cola');

cycola(cy, webcola); // error cola.adaptor is not a function
```

 I noticed that the package.json says main is WebCola/index.js, so this would essentially be the object that I get from require('webcola'), but the WebCola/cola.js seems to be maybe the file with the cytoscape adaptor function(?)

 I can pull in the cola.v3.min.js via a `<script>` tag which works fine, but I was curious about the npm setup

**Comment by maxkfranz** [comment]
It looks as though there is another build just for npm.

I'm guessing there should be `///<reference path="./src/adaptor.ts"/>` in https://github.com/tgdwyer/WebCola/blob/master/WebCola/index.ts#L16

**Comment by cooleso** [comment]
cola import if you use webpack for build app:
`import cola from 'exports?cola!webcola/WebCola/cola';`

that`s works for me :)

**Comment by hsellou** [comment]
Hello,
I'm no familiar with this syntax `import x from 'exports?'`, where did this come from ?
But yeah, it seems to work better with `webcola/WebCola/cola` but the cola module is not exported in this script ...

**Comment by maxkfranz** [comment]
Pull request #189 should address this.

I've also put a workaround on the Cy.js extension side.  There is a (corrected) built version of Cola in the `cytoscape-cola` package, so you can just `let cola = require('cytoscape-cola/cola')`.  The `cola.js` file in the extension is monkey patched so it supports *both* the browser and npm `module` in the same file.

Aside: It might be simpler if this package just built one file and checked for `module`:

```js
// patch in conditional cola commonjs support
if( typeof module !== 'undefined' ){
  module.exports = cola;
}
```

**Comment by hsellou** [comment]
Thanks you, it works well with *cytoscape-cola* 1.4.3

**Comment by tgdwyer** [comment]
Thanks for diagnosing this issue.  #189 merged now.

**Comment by hsellou** [comment]
Thank you Tim & Max.
@tgdwyer: can you push a release including this patch please ?

**Comment by tgdwyer** [comment]
npm published as webcola@3.2.0
